debugger/pythonExtension/gdb_sgx_plugin.py

```diff
@@ -246,9 +249,48 @@ def stop(self):
         update_untrusted_ocall_frame(frame_pointer, ocallcontext_tuple)
         return False
 
+def new_objfile_handler(event):
+    global g_enclave_list_parsed
+    if not g_enclave_list_parsed:
+        list_head = None        
+        try:
+            list_head = gdb.parse_and_eval("oe_enclave_list_head")            
+        except:
+            pass
+        enclaves = []
+        try:
+            if list_head != None:
+                print ("oe-gdb: Found global enclave list.")
+                # Set the 
+                g_enclave_list_parsed = True
+                node_ptr = list_head['lh_first']
+                while node_ptr != 0:
+                    node = node_ptr.dereference()
+                    enclave = node['enclave']
+                    enclave_addr = int(enclave)
+                    enclave_path = enclave.dereference()['path'].string('utf-8')
+                    enclaves.append((enclave_addr, enclave_path))
+                    node_ptr = node['next_entry']['le_next']
+                g_enclave_list_parsed = True;
```

Comment:
The ideal place is 274. 
We need to set the flag before calling enable_oeenclave_debug because that function loads the enclave binary to get symbols, triggering a recursive call to new_objfile_handler.

Reply:
ok